proto/utxorpc/v1alpha/cardano/cardano.proto

```diff
@@ -596,6 +598,25 @@ message PParams {
   uint64 drep_inactivity_period = 31; // The drep inactivity period.
 }
 
+message EraBoundary {
+  uint64 time = 1; // ms timestamp
+  uint64 slot = 2; // absolute slot number of the first block of this era
+  uint64 epoch = 3; // first epoch for this era
+  optional bytes hash = 4; // block hash of the first block of this era (if the era has started)
+}
+message EraSummary {
+  string name = 1; // name of the era (ex: "shelley")
+  EraBoundary start = 2; // start of this era
+  EraBoundary end = 3; // end of this era (if the era has a well-defined ending)
+}
+message EraSummaries {
+  repeated EraSummary summaries = 1;
+}
+
+message EraConfig {
+   google.protobuf.Struct config = 1;
```

Comment:
Here I used the `Struct` type since one of the recommended ways to deal with JSON (which Cardano genesis files use). However, the generated code for this is [pretty verbose](https://github.com/bufbuild/protobuf-es/blob/6a81dc6f398f44ba1c6f7fb7c48a0856ed07566d/packages/protobuf/src/wkt/gen/google/protobuf/struct_pb.ts#L84-L139) (it's not just a JSON object)

The other alternative is to just make the type be `string` (then people can use it with a JSON parser of their choice). I'm open to either option